pkg/onmetal/types.go

```diff
@@ -43,6 +43,16 @@ const (
 	// MachineControllerManagerProviderOnmetalImageName is the name of the MachineController onmetal image.
 	MachineControllerManagerProviderOnmetalImageName = "machine-controller-manager-provider-onmetal"
 
+	// AccessKeyID is a constant for the key in a onmetal bucket access secret that holds the Bucket access key id.
+	BucketAccessKeyID = "AWS_ACCESS_KEY_ID"
+	// AwsSecretAccessKey is a constant for the key in a onmetal bucket access secret that holds the Bucket secret access key.
+	BucketSecretAccessKey = "AWS_SECRET_ACCESS_KEY"
```

Comment:
@lukasfrank do we have somewhere constants for those access secret keys? Or are those coming from `ceph` directly?